src/components/common/StyledTextField/styles.js

```diff
@@ -1,10 +1,7 @@
 export const useStyles = theme => ({
   styledTextField: {
-    "& label": {
-      color: theme.palette.text.black1,
-    },
-    "& div": {
-      color: theme.palette.text.black1,
-    },
+    "& label": { color: theme.palette.text.black1 },
+    "& div": { color: "#212121 !important" },
+    "& fieldset": { borderColor: "#828282 !important" },
```

Comment:
Do we need `!important`?

Reply:
Yh. I confirmed it with Ananya.

Reply:
[raised issue](https://github.com/singnet/refactored-dapp/issues/39)